Library/Formula/spaceman-diff.rb

```diff
@@ -0,0 +1,44 @@
+class SpacemanDiff < Formula
+  desc "Diff images from the command line"
+  homepage "https://github.com/holman/spaceman-diff"
+  url "https://github.com/holman/spaceman-diff/archive/v1.0.tar.gz"
+  sha256 "934f38ae863abf15987100062268bcc28d37c39fe8c0a00a0813f07551944632"
+
+  head "https://github.com/holman/spaceman-diff.git"
+
+  depends_on "jp2a"
+  depends_on "imagemagick"
+  depends_on "roundup"
+
+  bottle :unneeded
+
+  def install
+    bin.install "spaceman-diff"
+
+    pkgshare.install "test"
+    pkgshare.install "tests"
+    pkgshare.install "spaceman-diff-test.sh"
+  end
+
+  def caveats; <<-EOS.undent
+    To finish the installation, create ~/.config/git/attributes and add:
+      *.png  diff=spaceman-diff
+      *.jpg  diff=spaceman-diff
+      *.jpeg diff=spaceman-diff
+      *.gif  diff=spaceman-diff
+
+    Then edit ~/.gitconfig and add:
+
+      [diff "spaceman-diff"]
+        command = #{HOMEBREW_PREFIX}/bin/spaceman-diff
+    EOS
+  end
+
+  test do
+    cp bin/"spaceman-diff", testpath
+    cp pkgshare/"tests", testpath
+    cp_r pkgshare/"test", testpath
+    cp_r pkgshare/"spaceman-diff-test.sh", testpath
+    system "./tests"
```

Comment:
Where should I get the images from? Do I add images somewhere into the homebrew repository?